Explicitly link runtime against zlib to fix "DSO missing from command line" error

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 0a7a6a09..331ecf1e 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -238,6 +238,8 @@ elseif(STATIC_BPF_BCC)
   target_link_libraries(runtime "-lz" "-llzma")
 endif()

+target_link_libraries(runtime "-lz")
+
 unset(MAIN_SRC)
 unset(BPFTRACE)
